example/README.md

```diff
@@ -2,27 +2,28 @@
 
 ## Overview
 
-The example shows one of the important Registry use-cases — services advertisement and discovery. So it is about advertisement by providers, discovery and usage by clients with the same Aqua code transparently without any knowledge about particular peer and service ids.
+The example shows one of the important Registry use-cases — services advertisement and discovery. So it is about advertisement, discovery and usage by clients with the same Aqua code transparently without any knowledge about particular peer and service ids.
```

Comment:
that doesn't sound right. first, service addr params need to be known for registration and Aqua also uses them. so the argument, i guess is, that the aqua function developer doesn't need to explicitly specify service addrs if a registry record is available.